packages/chrome/src/styled/sheet/StyledSheetClose.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme, css } from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.sheet_close';
+
+const colorStyles = (props: ThemeProps<DefaultTheme>) => {
+  const backgroundColor = 'primaryHue';
+  const foregroundColor = 'neutralHue';
+
+  return css`
+    background-color: transparent;
+    color: ${getColor(foregroundColor, 600, props.theme)};
+
+    &:hover {
+      background-color: ${getColor(backgroundColor, 600, props.theme, 0.08)};
+      color: ${getColor(foregroundColor, 700, props.theme)};
+    }
+
+    &[data-garden-focus-visible] {
+      box-shadow: ${props.theme.shadows.md(
+        getColor(backgroundColor, 600, props.theme, 0.35) as string
+      )};
+    }
+
+    &:active {
+      /* prettier-ignore */
+      transition:
+        background-color 0.1s ease-in-out,
+        color 0.1s ease-in-out;
```

Comment:
`props.isAnimated` check?

Reply:
Fair question. Garden's `isAnimated` prop is used to control potentially expensive macro animations. These micro (button hover, etc) animations happen as a matter of course and intended by design.